src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -259,12 +259,14 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
             dimensions: this.getDimensions(mdObject),
         };
 
-        const hasSizeMeasure = mdObject.buckets.some(bucket => bucket.localIdentifier === BucketNames.SIZE);
+        const hasSegmentAttribute = mdObject.buckets.some(
+            bucket => bucket.localIdentifier === BucketNames.SEGMENT,
```

Comment:
please add type and return type

Reply:
fixed

Reply:
I meant 
`(bucket: ABC): boolean => bucket.localIdentifier === BucketNames.SEGMENT `